infra/azure/main.tf

```diff
@@ -356,6 +375,12 @@ resource "azurerm_storage_management_policy" "test" {
   }
 }
 
+resource "azurerm_role_assignment" "test_test_container_contributor" {
```

Comment:
hmm. I think I find these easier to read when grouped by principal rather than by role. What do you think?